mandatory/debugging-practice/script.js

```diff
@@ -20,25 +20,26 @@ function populateStorage() {
   }
 }
 
-const title = document.getElementById("title");
-const author = document.getElementById("author");
-const pages = document.getElementById("pages");
-const check = document.getElementById("check");
-
 //check the right input from forms and if its ok -> add the new book (object in array)
 //via Book function and start render function
 function submit() {
+  const title = document.getElementById("title");
+  const author = document.getElementById("author");
+  const pages = document.getElementById("pages");
+  const check = document.getElementById("check");
+  //console.log(title);
   if (
-    title.value == null ||
-    title.value == "" ||
+    !title.value == null ||
```

Comment:
Keep using a consistent style of logical comparison on all conditions to improve code readability.